src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -417,40 +358,25 @@ Domain<3> BinaryCompactObject::create_domain() const {
 
   // --- Frustums enclosing both objects (10 blocks) ---
   //
-  // The two abutting cubes are enclosed by a layer of frustums that form a cube
-  // (if frustum_sphericity_ is 0) or a sphere (if frustum_sphericity_ is 1)
-  // surrounding both objects. While the two objects can be offset from the
-  // origin to account for their center of mass, the enclosing frustums are
+  // The two abutting cubes are enclosed by a layer of frustums that form a
+  // sphere surrounding both objects. While the two objects can be offset from
```

Comment:
can change one of `surrounding` or `enclosing` to `enveloping`. I think echoing the language used in the parameter names in the documentation too helps in making the parameter names clearer